src/lib.rs

```diff
@@ -1178,12 +1178,10 @@ unsafe fn close_internal(c_out: &mut *mut Rav1dContext, flush: c_int) {
         }
         free((*c).frame_thread.out_delayed as *mut c_void);
     }
-    let mut n_3 = 0;
-    while n_3 < (*c).n_tile_data {
-        rav1d_data_unref_internal(&mut (*((*c).tile).offset(n_3 as isize)).data);
-        n_3 += 1;
+    for mut tile in (*c).tiles.drain(..) {
+        rav1d_data_unref_internal(&mut tile.data);
     }
-    free((*c).tile as *mut c_void);
+    (*c).tiles = Default::default();
```

Comment:
Same as above, we should remove when context is owned.

Reply:
Done in 03c4d28.